src/utils/embed.ts

```diff
@@ -0,0 +1,8 @@
+export const detectEmbedded = () => {
+    const paramsString = window.location.search;
+    const searchParams = new URLSearchParams(paramsString);
+    if (searchParams.has("embed") && searchParams.get("embed") === "1") {
+        return true;
+    }
```

Comment:
Why `1` and not `true`?

Reply:
i guess convention? it is usually like this in urls you do 0 or 1  `?embed=1`